### DISCUSSION TITLE
resources error during JSBSimForUnreal.sln  build

### DISCUSSION THREAD

**Discussion by ahmadmasoud** [JSBSim-Team/jsbsim#680] [Q&A]
I have an issue with the build file  "JSBSimForUnreal.sln" as per image

![177371865-8fd6d6cc-2f03-4068-8b3d-9c960cba80c0](https://user-images.githubusercontent.com/47717125/177394129-8b1889a7-d095-40f0-b5ad-db9b952528ed.png)

**Comment by omaralmasoud** [comment]
It's sames like Post-build issue
you can try open JSBSimForUnreal.sln and put Post-build event to no
and copy files manually

`mkdir UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Source\ThirdParty\JSBSim\Include
robocopy $(SolutionDir)src UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Source\ThirdParty\JSBSim\Include *.h *.hxx /E
echo supresses errors 1, 2, and 4 which are not really errors.
set/A errlev="%ERRORLEVEL% &amp; 24"

echo Copy Aircrafts, Engines and Systems in the plugin resources folder
robocopy $(SolutionDir)aircraft UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Resources\JSBSim\aircraft /E
robocopy $(SolutionDir)engine UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Resources\JSBSim\engine /E
robocopy $(SolutionDir)systems UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Resources\JSBSim\systems /E

exit/B %errlev%` 

mkdir = make folder "Include"
robocopy = on repo file copy src content to UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Source\ThirdParty\JSBSim\Include
and finally on your repo file copy the following :
aircraft ,engine ,systems to UnrealEngine\Plugins\JSBSimFlightDynamicsModel\Resources\JSBSim
note : may  "JSBSim" folder not there just create it

hope that fix your issue now
